html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML forms
+    </title>
+  </head>
+  <body>
+    <h1><strong>Coding Form!</strong></h1>
+    <form>
+      <div><label for="email">Email:</label></div>
+      <div><input type="text" id="email" name="email" size="30"></div>
+      <div><label for="option">Why is coding cool?</label></div>
+      <div>
+        <select name="Why is coding cool?" id="Why is coding cool?">
+          <option value="Select an option">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <p>Favorite Language (choose one)</p>
+      </div>
+      <div>
+        <input type="radio" name="Favorite Language" value="HTML"><label>HTML</label>
+        <input type="radio" name="Favorite Language" value="CSS"><label>CSS</label>
+        <input type="radio" name="Favorite Language" value="JavaScript"><label>JavaScript</label>
```

Comment:
These `label` elements need to be properly configured to match with their paired `input` element.